www/templates/control/main-control.html

```diff
@@ -120,6 +126,13 @@
         <div class="control-list-text" ng-i18next>Dummy Notification in 5 Seconds</div>
         <div ng-click="dummyNotification()" class="control-icon-button"><i class="ion-ios-bell"></i></div>
       </div>
+      <!-- maybe notif schedule preview could go here? -->
+      <div class = "control-list-item">
+        <div class="control-list-text" ng-i18next>{Upcoming Notifications}</div>
```

Comment:
we were avoiding spaces and full text strings for the i18n keys before this. Let's discuss if we should change that.

By default, we shouldn't change it